### ISSUE TITLE
REST or other such API for data upload?

### ISSUE THREAD

**Issue by gdevenyi** [aces/Loris#3022]
Data upload (particularly imaging data) would be particular onerous task if many scans were collected per day.

If file-naming (and DICOM header data) is enforced for metadata, a web api would be useful for automated uploading.

Are there any plans to add such an API?

**Comment by johnsaigle** [comment]
Yes! As of now we don't have a timeline or person assigned to this task but it's absolutely something that's on our [roadmap](https://github.com/aces/Loris/projects/9).

**Comment by dyt811** [comment]
We sort of got one working but had to incorporate some backend update to the LORIS 0.0.3-dev APIs  and permission changes, in addition to roll out our intermediate Orthanc server in addition to a python program for monitoring/anonymization/upload/store identifiable information locally.  I was chatting with the representative from Douglas who was trying to get LORIS working. But whenever you guys ready to give it a try, feel free to reach out to us. 

We are deploying this across clinical scanners anyway so research scanners are far easier to handle.
